packages/material-ui-codemod/src/v5.0.0/moved-lab-components.js

```diff
@@ -0,0 +1,64 @@
+const movedLabComponents = [
+  'Alert',
+  'Autocomplete',
```

Comment:
```suggestion
  'AlertTitle',
  'Autocomplete',
  'useAutocomplete',
  'AvatarGroup',  
```

Reply:
Done. Renamed to `moved-lab-modules` accordingly